xmtp_mls/src/groups/mod.rs

```diff
@@ -272,6 +272,15 @@ where
         account_addresses_to_add: Vec<String>,
     ) -> Result<(), GroupError> {
         let account_addresses = sanitize_evm_addresses(account_addresses_to_add)?;
+        // get current number of users in group
+        let member_count = self.members().unwrap().len();
+        if member_count + account_addresses.len() > MAX_GROUP_SIZE as usize {
+            return Err(GroupError::Generic(format!(
```

Comment:
It probably makes sense to add a variant here rather than using `Generic`, seeing as this is the type of thing that might need to be programmatically detected and handled.